### ISSUE TITLE
"Invalid cursor provided" Error for "get_events()"

### ISSUE THREAD

**Issue by hhissami** [klaviyo/klaviyo-api-python#9]
Hi! 

Sometimes I get the following error message when calling **get_events()** with a cursor URL:

```
Status Code: 400
Reason: Bad Request
...
"status":400,"code":"invalid","title":"Invalid input.","detail":"Invalid cursor provided.","source":{"pointer":"/data/"}}]}
```

When I input this cursor URL to Postman, I get a normal response.

This error seems to occur more with older data.

Thanks in advance!

**Comment by jon-batscha** [comment]
Hey! Just shipped a new version 1.2.1, with the fix. 

Apologies for the wait. Mostly though, thank you so much for catching this bug that went unnoticed by many eyes, we super appreciate it.

If you run into any other issues, please do not hesitate to reach out.


Cheers!
Jon

**Comment by hhissami** [comment]
Thanks a lot Jon! 😀
